sql/ksp/_ksp.sql

```diff
@@ -33,8 +33,8 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 --v2.6
 -- CREATE FUNCTION _pgr_ksp(
 --     edges_sql TEXT,
---     start_vid INTEGER,
---     end_vid INTEGER,
+--     start_vid BIGINT,
```

Comment:
You have the function in comments, that is you removed the signature.
Put it back.
You can not remove a signature